chain/events/filter/index.go

```diff
@@ -75,20 +79,22 @@ var ddls = []string{
 	`INSERT OR IGNORE INTO _meta (version) VALUES (3)`,
 	`INSERT OR IGNORE INTO _meta (version) VALUES (4)`,
 	`INSERT OR IGNORE INTO _meta (version) VALUES (5)`,
+	`INSERT OR IGNORE INTO _meta (version) VALUES (6)`,
 }
 
 var (
 	log = logging.Logger("filter")
 )
 
 const (
-	schemaVersion = 5
+	schemaVersion = 6
 
 	eventExists          = `SELECT MAX(id) FROM event WHERE height=? AND tipset_key=? AND tipset_key_cid=? AND emitter_addr=? AND event_index=? AND message_cid=? AND message_index=?`
 	insertEvent          = `INSERT OR IGNORE INTO event(height, tipset_key, tipset_key_cid, emitter_addr, event_index, message_cid, message_index, reverted) VALUES(?, ?, ?, ?, ?, ?, ?, ?)`
 	insertEntry          = `INSERT OR IGNORE INTO event_entry(event_id, indexed, flags, key, codec, value) VALUES(?, ?, ?, ?, ?, ?)`
 	revertEventsInTipset = `UPDATE event SET reverted=true WHERE height=? AND tipset_key=?`
 	restoreEvent         = `UPDATE event SET reverted=false WHERE height=? AND tipset_key=? AND tipset_key_cid=? AND emitter_addr=? AND event_index=? AND message_cid=? AND message_index=?`
+	insertEventsSeen     = `INSERT OR IGNORE INTO events_seen(height, tipset_key, tipset_key_cid) VALUES(?, ?, ?)`
```

Comment:
Any reason we're storing both the tipset key and cid? I guess this applies to the events table as well.

Reply:
This is a good question for the rest of the schema actually, and @aarshkshah1992 is just conforming to what's there - we keep both in `event`, we insert them both with new events and we query matching them both for "exists" and when we do a "restore" operation. Maybe we should just ditch tipset_key and only ever use tipset_key_cid? I'm not seeing what value-add both are giving us in here.

Reply:
@aarshkshah1992 can you look at the possibility of removing the key and just using the CID for this particular case, we can deal with the other instances later I think.

Reply:
Changed this to just use the CID. Whenever we have they key, we also have the CID ofcourse.

Reply:
This is done.